package-lock.json

```diff
@@ -1,8 +1,3226 @@
 {
   "name": "azure-rest-api-specs-tests",
   "version": "0.1.0",
```

Comment:
please don't include this file in PR

Reply:
I have reverted to the master file. Thank you!